addons/mail/static/tests/message/message.test.js

```diff
@@ -1415,8 +1415,26 @@ test("Chat with partner should be opened after clicking on their mention", async
     await contains(".o-mail-Composer-input", { value: "@Test Partner " });
     await click(".o-mail-Composer-send:enabled");
     await click(".o_mail_redirect");
-    await contains(".o-mail-ChatWindow .o-mail-Thread");
-    await contains(".o-mail-ChatWindow", { text: "Test Partner" });
+    await contains(".o_avatar_card .o_card_user_infos", { text: "Test Partner" });
+    await contains(".o_card_user_infos", { text: "testpartner@odoo.com" });
+});
+
+test("Avatar Card of the partner should be opened after clicking on their mention without corresponding user", async () => {
```

Comment:
This test case could be done in above test case. So, the title would be something like `Avatar Card of the partner should be opened after clicking on their mention`